src/Containers/ConnectionDetails.js

```diff
@@ -26,6 +27,13 @@ export const ConnectionDetails = (id) => {
             },
             body: JSON.stringify(data)
         });
+
+        dispatch(
+            addNotification({
+                variant: 'success',
+                title: 'Sending Ping'
+            })
```

Comment:
looks dope. I'd probably also add a failure condition in case you can't hit the `/ping` endpoint for some reason in the fetch catch